src/styles/variables.css

```diff
@@ -1,65 +1,106 @@
+/* !!! REMOVED --secondary: rgb(0, 230, 196) & replaced with primary; used only in `FileUpload`,
```

Comment:
I'd say remove these comments, and when anyone runs into issues with color variables, just point them back at this PR so they understand what changed.